scRNA-seq/post-processing_scRNA-seq.Rmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "scRNA-seq_post_processing"
+output: html_notebook
+---
+
+**CCDL 2019**
+
+In this notebook, we'll perform quality control analyses and normalization of 
+scRNA-seq count data. 
+
+As opposed to bulk rna-seq, there are there are a few main things to look out for
+in single cell rna-seq:
+
+**Single-cell rna-seq has...**  
+- More amplification (and more of it's associated biases and error)  
+- More 0's in the gene expression data (most genes aren't expressed across cell types)  
+
+This script is adapted from [Seurat pbmc tutorial](https://satijalab.org/seurat/pbmc3k_tutorial.html) 
+and from [COMBINE lab's alevin tutorial](https://combine-lab.github.io/alevin-tutorial/2018/alevin-seurat/)
+
+For these analyses, we will need Seurat and dplyr packages. 
+```{r Install Packages}
+# Attach library
+library(Seurat)
```

Comment:
My general thought is that if I'm using one package throughout the script and using it all the time, then I just load it. Because this script is pretty Seurat- heavy, I went ahead and loaded it.

Reply:
I can be convinced otherwise though.